.github/workflows/build_and_deploy.yaml

```diff
@@ -56,20 +56,44 @@ jobs:
             CF_R2_ACCESS_KEY_ID: ${{ secrets.CF_R2_ACCESS_KEY_ID }}
             CF_R2_TOKEN: ${{ secrets.CF_R2_TOKEN }}
 
+    prepare-windows-ia32:
```

Comment:
I don't see this job ever referenced?

The two runs of `build_prepare` will both upload an artifact called `webapp` and conflict with each other